dev-libs/zziplib/zziplib-0.13.69.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit flag-o-matic libtool python-any-r1
+
+DESCRIPTION="Lightweight library for extracting data from files archived in a single zip file"
+HOMEPAGE="https://github.com/gdraheim/zziplib"
+SRC_URI="https://github.com/gdraheim/zziplib/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( LGPL-2.1 MPL-1.1 )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="doc sdl static-libs"
+
+RDEPEND="
+	sys-libs/zlib
+	sdl? ( >=media-libs/libsdl-1.2.6 )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}"
+BDEPEND="
+	virtual/pkgconfig"
+
+RESTRICT="test"
+
+src_prepare() {
+	python_fix_shebang .
+
+	# workaround AX_CREATE_PKGCONFIG_INFO bug #353195
+	sed -i \
+		-e '/ax_create_pkgconfig_ldflags/s:$LDFLAGS::' \
+		-e '/ax_create_pkgconfig_cppflags/s:$CPPFLAGS::' \
+		configure || die
+
+	# zziplib tries to install backwards compat symlinks we dont want
+	sed -i -e '/^zzip-postinstall:/s|$|\ndisable-this:|' Makefile.in || die
+	sed -i -e '/^install-exec-hook:/s|$|\ndisable-this:|' zzip/Makefile.in || die
+
+	elibtoolize
+
+	# Do an out-of-tree build as their configure will do it automatically
+	# otherwise and that can lead to funky errors. #492816
+	mkdir -p build || die
+
+	eapply_user
+}
+
+src_configure() {
+	cd "${S}"/build || die
+
+	append-flags -fno-strict-aliasing # bug reported upstream
+	export ac_cv_path_XMLTO= # man pages are bundled in .tar's
+
+	# Disable aclocal probing as the default path works #449156
+	ECONF_SOURCE=${S} \
+	ACLOCAL=true \
+	econf \
+		$(use_enable sdl) \
+		$(use_enable static-libs static)
+	MAKEOPTS+=' -C build'
+}
+
+src_install() {
+	default
+	# fowners fails when we don't have enough permissions (Prefix)
+	if [[ ${EUID} == 0 ]] ; then
+		fowners -R root /usr/share/man #321975
+	fi
+
+	find "${D}" -name '*.la' -delete || die
+
+	if use doc ; then
+		dohtml -r docs/*
```

Comment:
`dontml` is obsolete, and also you do not need globs when using something with the `-r` option

Reply:
Should be fixed on b75523f